sdkit/models/model_loader/hypernetwork/hypernetwork.py

```diff
@@ -56,7 +95,7 @@ def __init__(self, dim, state_dict=None, layer_structure=None, activation_func=N
 
         self.to(device)
 
-    def fix_old_state_dict(self, state_dict):
+    def fix_old_state_dict(self, state_dict) -> None:
```

Comment:
Again, I'm not a fan of the `-> None` declaration. It means nothing, unless the function really returned something meaningful. And this function intuitively doesn't return anything. The code doesn't need to be more verbose than necessary.